demo/src/main/java/com/novoda/merlin/demo/presentation/RxJava2DemoActivity.java

```diff
@@ -27,8 +27,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main);
         viewToAttachDisplayerTo = findViewById(R.id.displayerAttachableView);
-        networkStatusDisplayer = new NetworkStatusDisplayer(getResources(), merlinsBeard);
         merlinsBeard = MerlinsBeard.from(this);
+        networkStatusDisplayer = new NetworkStatusDisplayer(getResources(), merlinsBeard);
```

Comment:
Maybe add a `@NonNull` annotation to the `NetworkStatusDisplayer` so we're protected against this the next times?